keyboards/preonic/keymaps/egstad/keymap.c

```diff
@@ -0,0 +1,316 @@
+   1. To create a layer, define a (readable) name and set layer value
+   2. Under "Layer definitions" section, define keymaps
+   ========================================================================== */
+
+#define L_QWERTY  0 
+#define L_LOWER   1 
+#define L_RAISE   2
+#define L_ADJUST  3  
+
+
+
+
+/* TAP DANCE
+   ==========================================================================
+   1. Define TD names here
+   2. Register action in tap_dance_actions[]
+   ========================================================================== */
+
+enum {
+  TD_BRC = 0,
+  TD_MIN,
+  TD_GV_ESC,
+  TD_BS
+};
+
+
+
+
+/* CUSTOM KEYMAPS
+   ========================================================================== */
+
+// Tap dances
+#define TD_ESCP TD(TD_GV_ESC)           // Tap for grave, twice for escape
+#define TD_MINS TD(TD_MIN)              // Tap for minus, twice for equal
+#define TD_BRAC TD(TD_BRC)              // Tap for open brace, twice for close
+
+// Layers
+#define LT2_TAB LT(L_RAISE, KC_TAB)     // Tap for tab, hold for RAISE
+#define LT2_ENT LT(L_RAISE, KC_ENT)     // Tap for tab, hold for RAISE
+#define LT1_BSP LT(L_LOWER, KC_BSPACE)  // Tap for backspace, hold for LOWER
+#define LT1_SPC LT(L_LOWER, KC_SPC)     // Tap for space, hold for LOWER
+
+// Modifiers
+#define MT_SHFT MT(MOD_RSFT, KC_ENT)    // Tap for enter, hold for shift
+#define MT_HILF S(A(KC_LEFT))           // Press for shift + alt + left
+#define MT_HIRT S(A(KC_RGHT))           // Press for shift + alt + right
+#define MT_UNTB S(KC_TAB)               // Press for shift + tab
+
+
+
+
+
+
+/* Layer definitions
+   ========================================================================== */
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* QWERTY
+ * ,-----------------------------------------------------------------------------------------------.
+ * | ` ESC |  1 !  |  2 @  |  3 #  |  4 $  |  5 %  |  6 ^  |  7 &  |  8 *  |  9 (  |  0 )  | -_ =+ |
+ * |-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------|
+ * | TB/RS |   Q   |   W   |   E   |   R   |   T   |   Y   |   U   |   I   |   O   |   P   | [{ }] | 
+ * |-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------|
+ * | BS/LW |   A   |   S   |   D   |   F   |   G   |   H   |   J   |   K   |   L   |   ;   |   '   |
+ * |-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------|
+ * |  SFT  |   Z   |   X   |   C   |   V   |   B   |   N   |   M   |   ,   |   .   |   /   | SF/RN |
+ * |-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------+-------|
+ * | LEADR |       |  CTL  |  ALT  |  CMD  |  BSP  | SP/LW | RN/RS |   ←   |   ↑   |   ↓   |   →   |    
+ * `-----------------------------------------------------------------------------------------------
+ */
+
+[L_QWERTY] = LAYOUT_preonic_grid( \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.